src/components/QueryBuilder.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<div class="querybuilder">
+		<h1 class="querybuilder__heading">Simple Query Builder</h1>
+		<p class="querybuilder__description">
+			Short explanatory text that also manages expectations. Short explanatory text that also
+			manages expectations.
+		</p>
+		<h4 class="querybuilder__find-title">Find all items...</h4>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend( {
+	name: 'QueryBuilder'
+} );
+</script>
+
+<style scoped lang="scss">
+@import '~@wmde/wikit-tokens/dist/variables';
+
+.querybuilder {
+	position: relative;
+	width: 1024px;
+	height: 349px;
```

Comment:
```suggestion
```

Those aren't part of the specs are far as I can tell. If I'm not mistaken the box should just grow with the window size.

Reply:
Yeah, I copied it from figma but probably I don't know how to copy things properly